src/lxc/tools/lxc_attach.c

```diff
@@ -36,7 +36,7 @@
 #include <lxc/lxccontainer.h>
 
 #include "attach.h"
-#include "arguments.h"
+#include "tools/arguments.h"
```

Comment:
That's actually unnecessary you should just include `"arguments.h"`. What I meant in the last comment is what you did in `Makefile.am` above, namely prefixing the references to `arguments.{c,h}` for each tool with `tools/`.
The benefit of doing it just in the Makefile and not in each binary is that you have a single place to change when you decide to change paths. For example, when you were to move everything from `tools/` to `cmd/` then you would only need to change `Makefile.am` and not each binary.